tests/qam-minimal/install_update.pm

```diff
@@ -32,7 +30,7 @@ sub run {
 
     zypper_call("ref");
 
-    zypper_call(qq{in -l -y -t patch \$(zypper patches | awk -F "|" '/test-minimal/ { print \$2;}')}, exitcode => [0, 102, 103], log => 'zypper.log');
+    zypper_call('in -l -t patch ' . get_var('INCIDENT_PATCH'), exitcode => [0, 102, 103], log => 'zypper.log');
```

Comment:
ok, I don't think it is important which line is original. Can you extract a helper function to call this on both ocassions?